include/mbedtls/ssl.h

```diff
@@ -26,6 +26,8 @@
 
 #include "mbedtls/build_info.h"
 
+#include "hash_info.h"
```

Comment:
Ah, this was stupid: including a private header from a public one is not going to work.